src/nl/rubensten/texifyidea/settings/labeldefiningcommands/LabelCommandSettingsTable.kt

```diff
@@ -0,0 +1,27 @@
+package nl.rubensten.texifyidea.settings.labeldefiningcommands
+
+import com.intellij.ui.SimpleTextAttributes
+import com.intellij.ui.table.JBTable
+import com.intellij.util.ui.StatusText
+import java.awt.event.ActionListener
+import javax.swing.table.TableModel
+
+/**
+ * JBTable is extended to set own text when table is empty and define action
+ */
+class LabelCommandSettingsTable(model: TableModel, private var listener: ActionListener) : JBTable(model) {
+
+    /**
+     * create empty text and define a clickable link to add new command
+     */
+    override fun getEmptyText(): StatusText {
+        val test = object: StatusText(this) {
+            override fun isStatusVisible(): Boolean {
+                return isEmpty
+            }
+        }
+        test.text = "No command defined"
+        test.appendSecondaryText("Define new command", SimpleTextAttributes.LINK_ATTRIBUTES, listener)
```

Comment:
@PHPirates: Input on this?
![image](https://user-images.githubusercontent.com/17410729/56140457-409b4d00-5f9b-11e9-8dab-fea1779fe233.png)

Reply:
I don't get that inspection. In fact, both in plugin.xml and in build.gradle the minimum version is 173.* so I wouldn't worry about it. I don't know where the 141 comes from, possibly an old module something still wandering around on your system.